fastapi/encoders.py

```diff
@@ -96,7 +96,7 @@ def jsonable_encoder(
                     or (not key.startswith("_sa"))
                 )
                 and (value is not None or not exclude_none)
-                and ((include and key in include) or key not in exclude)
+                and ((include and key in include) or not exclude or key not in exclude)
```

Comment:
I think your code looks wrong. When "not exclude" is true , "key not in exclude " does not check.
```suggestion
                and ((include and key in include) or (not exclude and key not in exclude))
```